docs/src/user/installing-kubernetes.rst

```diff
@@ -0,0 +1,79 @@
+
+.. _`installing-kubernetes`:
+
+Installing Kubernetes
+====================================================================
+
+Usage of Rafiki in Kubernetes mode requires Kubernetes 1.15+. 
+
+To achieve this, we recommend the instructions below:
+
+    1. Install kubelet kubeadm kubectl
+
+        .. code-block:: shell
+
+            apt-get install -y kubelet kubeadm kubectl --allow-unauthenticated
```

Comment:
need to add to source first

```
curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add
```
and
```
sudo apt-add-repository "deb http://apt.kubernetes.io/ kubernetes-xenial main"
```
and
```
sudo apt-get update
```

Reply:
I forgot it ,sorry. I remenber I add Aliyun source first, but I think Aliyun is not suitable for other country except China.